packages/cli/src/commands/pg/credentials/rotate.ts

```diff
@@ -0,0 +1,82 @@
+import color from '@heroku-cli/color'
+import {Command, flags} from '@heroku-cli/command'
+import type {AddOnAttachment} from '@heroku-cli/schema'
+import {Args, ux} from '@oclif/core'
+import confirmApp from '../../../lib/apps/confirm-app'
+import {attachment} from '../../../lib/pg/fetcher'
+import host from '../../../lib/pg/host'
+import {legacyEssentialPlan} from '../../../lib/pg/util'
+
+export default class Rotate extends Command {
+    static topic = 'pg'
+    static description = 'rotate the database credentials'
+    static flags = {
+      name: flags.string({char: 'n', description: 'which credential to rotate (default credentials if not specified)'}),
+      all: flags.boolean({description: 'rotate all credentials'}),
+      confirm: flags.string({char: 'c'}),
+      force: flags.boolean({description: 'forces rotating the targeted credentials'}),
+      app: flags.app({required: true}),
+    }
+
+    static args = {
+      database: Args.string(),
+    }
+
+    public async run(): Promise<void> {
+      const {flags, args} = await this.parse(Rotate)
+      const {app, all, confirm, name, force} = flags
+      const {addon: db} = await attachment(this.heroku, app, args.database)
+      const warnings: string[] = []
+      const cred = name || 'default'
```

Comment:
We can set this as a default in the flag definition:
```
  name: flags.string({char: 'n', default: 'default' ... })
```

Reply:
+1